File: backend/components/metiers/administration_publique/templates/helpers/samples/sample_helper_template.test.py
# sample_helper_template.test.py – Test Python exemple helper template
from .sample_helper_template import sample_helper_template

def test_sample_helper_template_ok():
    assert sample_helper_template({'a': 1}) == {'a': 1, 'helper': True}
    assert sample_helper_template(None) == {'helper': True, 'empty': True}

File: backend/components/metiers/administration_publique/templates/helpers/samples/sample_helper_template.test.js
// sample_helper_template.test.js – Test JS exemple helper template
const { sampleHelperTemplate } = require('./sample_helper_template');
test('sampleHelperTemplate fonctionne', () => {
  expect(sampleHelperTemplate({ a: 1 })).toEqual({ a: 1, helper: true });
  expect(sampleHelperTemplate()).toEqual({ helper: true, empty: true });
});
